lib/ReplaceSource.js

```diff
@@ -18,16 +18,16 @@ class ReplaceSource extends Source {
 		this.replacements = [];
 	}
 
-	replace(start, end, newValue) {
+	replace(start, end, newValue, oldValue = undefined) {
```

Comment:
I suppose older versions of Node don't support default values. I'll replace this code.